source/manual/howto-switch-default-branch.html.md

```diff
@@ -0,0 +1,26 @@
+---
+owner_slack: "#govuk-developers"
+title: How to switch a git default branch from master to main
```

Comment:
```suggestion
title: How to change the default branch of a Git repo
```

It might be nice to keep the docs generic enough that they can stay around - but maybe that's unnecessary...?

Reply:
We'd need to update release to accept any input for default branch if we choose to do that. It's a pretty trivial UI change though. I decided against having it being free text as it seemed very unlikely to need other options.

Reply:
That's true, it's probably not worth it really.

Reply:
I'd still suggest capitalisation of `Git` and use the phrase `change` rather than `switch` though (but this is personal preference).

Reply:
Ok left it specific, but upper case G on Git.